documentation/asciidoc/computers/camera/rpicam_still.adoc

```diff
@@ -102,3 +102,100 @@ $ rpicam-still -o long_exposure.jpg --shutter 100000000 --gain 1 --awbgains 1,1
 ----
 
 To find the maximum exposure times of official Raspberry Pi cameras, see xref:../accessories/camera.adoc#hardware-specification[the camera hardware specification].
+
+==== Create a time lapse video
+
+To create a time lapse video, capture a still image at a regular interval, such as once a minute, then use an application to stitch the pictures together into a video.
+
+===== via `rpicam-still` time lapse mode
+
+To use the built-in time lapse mode of `rpicam-still`, use the xref:camera_software.adoc#timelapse[`timelapse`] option. This option accepts a value representing the period of time you want your Raspberry Pi to wait between captures, in milliseconds.
+
+First, create a directory where you can store your time lapse photos:
+
+[source,console]
+----
+$ mkdir timelapse
+----
+
+Run the following command to create a time lapse over 30 seconds, recording a photo every two seconds, saving output into `image0001.jpg` through `image0014.jpg`:
+
+[source,console]
+----
+$ rpicam-still --timeout 30000 --timelapse 2000 -o timelapse/image%04d.jpg
+----
+
+===== via `cron`
+
+You can also automate time lapses with `cron`. First, create the script, named `timelapse.sh` containing the following commands. Replace the `<username>` placeholder with the name of your user account on your Raspberry Pi:
+
+----
+#!/bin/bash
+DATE=$(date +"%Y-%m-%d_%H%M")
+rpicam-still -o /home/<username>/timelapse/$DATE.jpg
+----
+
+Then, make the script executable:
+
+[source,console]
+----
+$ chmod +x timelapse.sh
+----
+
+Create the `timelapse` directory into which you'll save time lapse pictures:
+
+[source,console]
+----
+$ mkdir timelapse
+----
+
+Open your crontab for editing:
+
+[source,console]
+----
+$ crontab -e
+----
+
+Once you have the file open in an editor, add the following line to schedule an image capture every minute, replacing the `<username>` placeholder with the username of your primary user account:
+
+----
+* * * * * /home/<username>/timelapse.sh 2>&1
+----
+
+Save and exit, and you should see this message:
+
+----
+crontab: installing new crontab
+----
+
+TIP: To stop recording images for the time lapse, remove the above line from your crontab.
+
+===== Stitch images together
+
+Once you have a series of time lapse photos, you probably want to combine them into a video. Use `ffmpeg` to do this on a Raspberry Pi.
+
+First, install `ffmpeg`:
+
+[source,console]
+----
+$ sudo apt install ffmpeg
+----
+
+Run the following command in the `crontimelapse` directory to convert your JPEG files into an mp4 video:
```

Comment:
crontimelapse -> timelapse ?

Reply:
Also, if you follow the instructions above, the images will be in `/home/<username>/timelapse/` and the command-line below uses `timelapse/*.jpg`, so it looks like it should probably just be run in your home directory rather than from any sub-directory?